#!/usr/bin/env python
PACKAGE = "imageproc_node"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("left_gain", double_t, 0, "Multiplier to Left Motor throttle", 1, -2, 2)
gen.add("right_gain", double_t, 0, "Multiplier to Right Motor throttle", 1, -2, 2)
gen.add("linear_gain", int_t, 0, "Linear throttle magnitude", 600, 0, 2400)
gen.add("angular_gain", int_t, 0, "Angular throttle magnitude", 600, 0, 2400)
gen.add("min_throttle", int_t, 0, "Throttle minimum", -4000, -4000, 0)
gen.add("max_throttle", int_t, 0, "Throttle maximum", 4000, 0, 4000)
exit(gen.generate(PACKAGE, "imageproc_node", "Drive"))
